package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "legacyframework",
+  "version": "4.0.0",
+  "description": "Legacy is a modern, lightweight CSS framework.",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/joexn/LegacyFramework.git"
+  },
+  "author": "joexn",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/joexn/LegacyFramework/issues"
+  },
+  "homepage": "https://github.com/joexn/LegacyFramework#readme",
+  "devDependencies": {
```

Comment:
In this case, it would be a terrible idea to check in the dependencies - no need to fill the repo with things that are not explicitly Legacy. That is partly why the package exists - so users can resolve dependencies for development if they choose to do so. I do agree on the latter point however, and will push a .gitignore for node_modules next commit.